src/requirementsFileExercise/requirements_file_exercise.py

```diff
@@ -1,32 +1,42 @@
 from typing import List
 
-from src.requirementsFileExercise.data_extractor import get_file_name, get_package, has_requirement_file_reference
+from src.requirementsFileExercise.package import Package
 from src.requirementsFileExercise.file_reader import read_file_lines
 
 
 def create_libraries_string(file_name: str) -> str:
```

Comment:
what will happens when there is in requirement_file_A: numpy == 2.0.0
and in requirement_file_B: numpy == 3.0.0?